gluecodium/src/main/java/com/here/gluecodium/generator/cpp/CppGeneratorSuite.kt

```diff
@@ -300,8 +307,10 @@ internal class CppGeneratorSuite(options: Gluecodium.Options) : GeneratorSuite {
     ): GeneratedFile {
         val templateData = mapOf("internalNamespace" to internalNamespace, "exportName" to exportName)
         val content = TemplateEngine.render("cpp/common/Export", templateData)
-        val namePrefix = Paths.get(GENERATOR_NAME, "include", internalNamespace.joinToString("/"), "Export").toString()
-        return GeneratedFile(content, "$namePrefix$infix.h", sourceSet)
+
+        val fileName = "Export" + infix + NameHelper.toUpperCamelCase(exportName)
```

Comment:
There are 3 very similar lines to construct this name, not sure if it's possible to improve though

Reply:
Done